Gemfile

```diff
@@ -2,6 +2,7 @@ source "https://rubygems.org"
 
 gemspec
 
+gem 'mime-types', '~> 2.99' if RUBY_VERSION < '2.0'
```

Comment:
See the [Gemfile now on master](https://github.com/sj26/mailcatcher/blob/master/Gemfile). :-)